anti-analysis/anti-av/patch-event-tracing-for-windows-function.yml

```diff
@@ -19,8 +21,8 @@ rule:
       - match: link function at runtime on Windows
       - or:
         - api: kernel32.VirtualProtect
-        - api: ntdll.NtProtectVirtualMemory  # exported by only ntdll, not ntoskrnl
```

Comment:
comments missing

Reply:
if these persist to be an annoying issue, maybe we don't upgrade them for now and leave for manual review?